src/propagation/propagation.jl

```diff
@@ -0,0 +1,462 @@
+
+See also [`__save2jldandcheck`](@ref). 
+"""
+function save2jldandcheck(objname, sol)
+    # Name of the file 
+    outfilename = string(objname, "_jt.jld2")
+    # Save sol in outfilename
+    return __save2jldandcheck(outfilename, sol)
+end
+
+@doc raw"""
+    __save2jldandcheck(outfilename, sol)
+
+Savs `sol` in `outfilename`. 
+"""
+function __save2jldandcheck(outfilename, sol)
+    println("Saving solution to file: $outfilename")
+    # Open file 
+    JLD2.jldopen(outfilename, "w") do file
+        # Loop over solution variables
+        for ind in eachindex(sol)
+            # Name of the variable 
+            varname = string(ind)
+            println("Saving variable: ", varname)
+            # Write the varaible 
+            write(file, varname, sol[ind])
+        end
+    end
+    # Check that saved solution is equal to the original 
+    println("Checking that all variables were saved correctly...")
+    # Loop over solution variables
+    for ind in eachindex(sol)
+        # Name of the variable 
+        varname = string(ind)
+        # Read varname from files and assign recovered variable to recovered_sol_i
+        recovered_sol_i = JLD2.load(outfilename, varname)
+        # Check that varname was recovered succesfully
+        @show recovered_sol_i == sol[ind]
+    end
+    println("Saved solution")
+    return outfilename
+end
+
+@doc raw"""
+    taylor_minimum(pol::Taylor1{T}, x0::T; niters::Int=10) where {T<:Real}
+
+Return the minimum of the Taylor polynomial `pol` computed via Newton's method. `x0` is the
+initial guess and `niters` is the number of iterations. 
+"""
+function taylor_minimum(pol::Taylor1{T}, x0::T; niters::Int=10) where {T<:Real}
+    # First derivative 
+    dpol = PlanetaryEphemeris.ordpres_differentiate(pol)
+    # Second derivative
+    dpol2 = PlanetaryEphemeris.ordpres_differentiate(dpol)
+    # Initial guess
+    xnewton::T = x0
+    #@show xnewton
+    # Newton iteration 
+    for i in 1:niters
+        # Newton update rule 
+        xnewton -= dpol(xnewton)/dpol2(xnewton)
+        #@show xnewton, dpol(xnewton)
+    end
+
+    return xnewton
+end
+
+@doc raw"""
+    taylor_roots(pol::Taylor1{T}, x0::T; niters::Int=10) where {T<:Real}
+
+Return the root of the Taylor polynomial `pol` computed via Newton's method. `x0` is the 
+initial guess and `niters` is the number of iterations. 
+"""
+function taylor_roots(pol::Taylor1{T}, x0::T; niters::Int=10) where {T<:Real}
+    # First derivative
+    dpol = PlanetaryEphemeris.ordpres_differentiate(pol)
+    # Initial guess
+    xnewton::T = x0
+    #@show xnewton
+    # Newton iteration  
+    for i in 1:niters
+        # Newton update rule 
+        xnewton -= pol(xnewton)/dpol(xnewton)
+        #@show xnewton, pol(xnewton)
+    end
+    return xnewton
+end
+
+@doc raw"""
+    scaling(a::Taylor1{Taylor1{T}}, c::T) where {T<:Real}
+
+Scale `a` by a factor `c`. 
+"""
+function scaling(a::Taylor1{Taylor1{T}}, c::T) where {T<:Real}
```

Comment:
Question: do we truly use nested `Taylor1`s? Or is it a left over of something we tested in the old past?

Reply:
So far I've not used this function, it might be a left over from currect NEOs master. @PerezHz could you explain the purpose of this function?

Reply:
Well, this function is (or dare I say, was once?) useful for scaling variables with 1-variable jet-transport, but it's a two-liner and currently not used in the code, so maybe we can just scrap it...

Reply:
Since this function won't have a use in the near future, I'll deprecate it.